### PR TITLE
chore: keep pkg-dir at <6

### PR DESCRIPTION
**Pull request by netlify-team-account-1**
as proposed in https://github.com/netlify/zip-it-and-ship-it/pull/697#issuecomment-933380898

### CODE REVIEW

**Review by erezrokah — APPROVED** [review]

File: default.json [hunk 1]
```diff
@@ -70,6 +70,10 @@
     {
       "matchPackageNames": ["junk"],
       "allowedVersions": "<4"
-    }
+    },
+    {
+      "matchPackageNames": ["pkg-dir"],
+      "allowedVersions": "<6",
+    },
```
Comment (erezrokah) [inline default.json:77]:
```suggestion
    }
```

**Review by erezrokah — CHANGES_REQUESTED** [review]
This needs to be a valid JSON

**Comment by erezrokah** [comment]
@netlify-team-account-1, you're too fast for me. I reviewed then noticed the PR breaks the JSON structure

**Comment by netlify-team-account-1** [comment]
😅 I'll open another PR, wait a sec

**Pull request merged**